travis-ymls/colors.travis.yml

```diff
@@ -0,0 +1,42 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : colors
+# Source Repo         : https://github.com/Marak/colors.js
+# Travis Job Link     : https://travis-ci.com/github/dthadi3/colors.js/builds/211018706
+# Created travis.yml  : <Yes, if travis.yml is created. No, if existing travis.yml is modified.>
```

Comment:
Please set this field to either "Yes" or "No" as appropriate.

Reply:
yeah updated suggested changes, and changed license to Apache.